pkg/cmd/admin/prune/images.go

```diff
@@ -99,6 +115,16 @@ func (o *PruneImagesOptions) Complete(f *clientcmd.Factory, cmd *cobra.Command,
 		return kcmdutil.UsageError(cmd, "no arguments are allowed to this command")
 	}
 
+	if !cmd.Flags().Lookup("keep-younger-than").Changed {
```

Comment:
Is `.Changed` `true` also when the flag is specified with default value? If not, can this check for a presence of the flag among `args` instead?